### ISSUE TITLE
`Sharing failed, please try again` for WhatsApp on mobile

### ISSUE THREAD

**Issue by pulsejet** [MurhafSousli/ngx-sharebuttons#242]
## Bug Report or Feature Request (mark with an `x`)
```
- [ x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?
Android Marshmallow 6.0

### Versions
Angular 5, latest ngx-sharebuttons

### Repro steps
Happens on the demo page https://murhafsousli.github.io/ngx-sharebuttons/#/share-button-directive
Sharing fails with `Sharing failed, please try again`. Also, on desktop, WhatsApp web opens, but there is no prompt to share the URL. Chrome on both. Other share buttons seem to work.

### Desired functionality
Should work

**Comment by MurhafSousli** [comment]
@pulsejet Can you try v5.2.0 and tell me if the problem still occurs

**Comment by pulsejet** [comment]
Fixed in v5.2.0
